src/core/annotation.js

```diff
@@ -1264,18 +1264,25 @@ class WidgetAnnotation extends Annotation {
     if (!annotationStorage || isPassword) {
       return null;
     }
-    const value =
+    let value =
       annotationStorage[this.data.id] && annotationStorage[this.data.id].value;
     if (value === undefined) {
       // The annotation hasn't been rendered so use the appearance
       return null;
     }
 
+    value = value.trim();
+
     if (value === "") {
       // the field is empty: nothing to render
       return "";
```

Comment:
A pre-existing issue, but why are we simply not returning `null` here if there's in fact nothing to render?

That's what we do in the `value === undefined` case just above, so what's special about *this* code-path?

Reply:
Returning `null` means that we'll use `AP::N` to render the field.
Here `value===""` means that the field is empty so it could be rendered using  `/Font 10 Tf 0 g () Tj` which is useless.